onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -6014,10 +6014,93 @@ def aten_var(self: TensorType, unbiased: bool = True) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_var_mean(self: TensorType, unbiased: bool = True) -> tuple[TensorType, TensorType]:
+@torch_op("aten::var_mean", trace_only=True)
+def aten_var_mean(self: TReal, unbiased: bool = True) -> tuple[TReal, TReal]:
     """var_mean(Tensor self, bool unbiased=True) -> (Tensor, Tensor)"""
 
-    raise NotImplementedError()
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean(self, correction, keepdim=False)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_dim(
+    self: TReal, dim: int, unbiased: bool = True, keepdim: bool = False
+) -> tuple[TReal, TReal]:
+    """var_mean.dim(Tensor self, int[1]? dim, bool unbiased=True, bool keepdim=False) -> (Tensor, Tensor)"""
+
+    if unbiased:
+        correction = 1
+    else:
+        correction = 0
+    var, mean = _aten_var_mean_dim(self, dim, correction, keepdim)
+    return var, mean
+
+
+@torch_op("aten::var_mean", overload=True, trace_only=True)
+def aten_var_mean_correction(
+    self: TReal, dim: int = None, correction: int = 1, keepdim: bool = False
+) -> tuple[TReal, TReal]:
+    """var_mean.correction(Tensor self, int[1]? dim=None, *, Scalar? correction=None, bool keepdim=False) -> (Tensor, Tensor)"""
+
+    if correction is None:
+        correction = 1
+
+    if dim is None:
+        var, mean = _aten_var_mean(self, correction, keepdim)
+    else:
+        var, mean = _aten_var_mean_dim(self, dim, correction, keepdim)
+    return var, mean
+
+
+@torch_op("aten::var_mean", private=True)
+def _aten_var_mean(
+    self: TReal, correction: int = 1, keepdim: bool = False
+) -> tuple[TReal, TReal]:
+    # Compute mean and var
+    mean = op.ReduceMean(self, keepdims=keepdim)
+    self_shape = op.Shape(self)
+    numel_int = op.ReduceProd(self_shape, keepdims=0)
+    sub_mean = op.Sub(self, mean)
+    sqr_mean = op.Mul(sub_mean, sub_mean)
+    var = op.ReduceMean(sqr_mean, keepdims=keepdim)
+    # Processing correction value appling to var
+    if correction != 0:
+        numel_float = op.Cast(numel_int, to=FLOAT.dtype)
+        mul = op.Mul(var, numel_float)
+        sub = op.Sub(numel_int, correction)
+        var = op.Div(mul, op.Cast(sub, to=FLOAT.dtype))
+
+    return var, mean
+
+
+@torch_op("aten::var_mean", private=True)
+def _aten_var_mean_dim(
+    self: TReal, dim: int, correction: int = 1, keepdim: bool = False
+) -> tuple[TReal, TReal]:
```

Comment:
```suggestion
) -> Tuple[TReal, TReal]:
```

for python3.8 compatibility